docs/.rstcheck.cfg

```diff
@@ -0,0 +1,4 @@
+[rstcheck]
+ignore_directives=automodule,http:get
+ignore_roles=djangosetting,setting
+ignore_messages=(Duplicate implicit target name: ".*")|(Hyperlink target ".*" is not referenced)
```

Comment:
Those errors are from some hyperlinks that are referenced globally. The duplicates are really false positives.

Reply:
By ignoring this now that are false positivies, we won't detect the ones that are true when we updated the docs :(

Reply:
Yeah, I going to try to solve others and see if I can get a better regex here

Reply:
There is a better way to expose global hiperlinks? Maybe something like this https://stackoverflow.com/a/19543591/5689214? And for the duplicates target name the most are from subheaders with the same name.

Reply:
Why not use the `:ref:` directive for linking across files?

http://www.sphinx-doc.org/es/stable/markup/inline.html#role-ref

Reply:
rstcheck doesn't search for global refs and got marked as unused. But using ref is fine.

Reply:
And also I change others place to use the `doc` directive https://docs.readthedocs.io/en/latest/docs.html#content.